src/pages/offer-details/OfferDetails.tsx

```diff
@@ -74,6 +78,10 @@ const OfferDetails = () => {
           onBookmarkClick={onBookmarkClick}
         />
       </AppCard>
+      <MultiAccordionWithTitle
```

Comment:
your component should be wrapped in appCard as other component on page, or styles should be passed to it. don`t use styles needed only for one page inside of reusable component

Reply:
okey, changed